task5/src/main/java/com/clouway/core/DataValidator.java

```diff
@@ -37,4 +37,15 @@ public boolean isValid(String username, String password, String email) {
     }
     return result;
   }
+
+  public boolean isValid(String amount) {
+    boolean result = false;
+    String amounPattern = "[0-9]{1,13}(\\.[0-9]*)?";
```

Comment:
I think that `2.565488756` will be valid. Make it validate amount with 2 decimal points as valid. all others are not.